src/lightning_app/core/flow.py

```diff
@@ -110,7 +110,18 @@ def name(self):
         """Return the current LightningFlow name."""
         return self._name or "root"
 
-    def __setattr__(self, name, value):
+    def _get_property_if_exists(self, name: str) -> Union[property, None]:
+        attr = getattr(self.__class__, name, None)
+        return attr if isinstance(attr, property) else None
+
+    def __setattr__(self, name: str, value: Any) -> None:
+        property_object = self._get_property_if_exists(name)
+        if property_object is not None and property_object.fset is not None:
+            property_object.fset(self, value)
+        else:
+            self._default_setattr(name, value)
+
+    def _default_setattr(self, name, value):
```

Comment:
This feels like it could be simplified, something like:
```suggestion
    def __setattr__(self, name: str, value: Any) -> None:
        attr = getattr(self.__class__, name, None)
        if isinstance(attr, property) and property_object.fset is not None:
            return property_object.fset(self, value)
```